client/src/App.js

```diff
@@ -1,10 +1,52 @@
 import React from 'react';
+import GlobalStyles from './GlobalStyles';
+import { ThemeProvider } from 'emotion-theming';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import primary from './themes/primary';
+import Header from './components/Header';
+import Footer from './components/Footer';
+import HashInsert from './components/HashInsertCard';
+import FormCard from './components/FormCard';
+import CategoryCard from './components/CategoryCard';
+import styled from '@emotion/styled';
+import PlayCard from './components/PlayCard';
+import AddQuestionsCard from './components/AddQuestionsCard';
+
+const Main = styled.main`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-flow: column nowrap;
+  height: 100vh;
+  width: 100vw;
+`;
 
 function App() {
   return (
-    <div>
-      <h1>Hallo Fische</h1>
-    </div>
+    <>
+      <ThemeProvider theme={primary}>
+        <Router>
+          <GlobalStyles />
+          <Main>
+            <Header />
```

Comment:
I think that header is normally not part of main.

Reply:
Thanks, I thought about it... my main is also my flex box and centring items doesn't work otherwise. Would have to create another wrapper then.